lib/sdt_task.bpf.c

```diff
@@ -391,7 +391,6 @@ static SDT_TASK_FN_ATTRS void sdt_task_free_idx(__u64 idx)
 
 		data[pos] = (struct sdt_task_data) {
 			.tid.gen = data->tid.gen + 1,
-			.tptr = 0,
```

Comment:
Why is `.tptr` being removed?

Reply:
We are currently not using it. I can instead properly populate it to make debugging easier for the task allocator (AFAICT that was its initial purpose, right)?

Reply:
IIRC, the intention was enabling easily looking up task struct from tid. How would you go from tid to task without the field?

Reply:
Ah I see, yeah the caller of the API passes ```struct task_struct *```  so they would already have the tptr available, but other readers of the struct like userspace wouldn't have a way of finding which task the data belongs to if reading it straight from the allocator.